core/scripts/cre/engine/utils.go

```diff
@@ -37,6 +39,16 @@ func NewStandaloneEngine(ctx context.Context, lggr logger.Logger, registry *capa
 		MaxCompressedBinarySize: defaultMaxUncompressedBinarySize,
 		IsUncompressed:          true,
 	}
+
+	module, err := host.NewModule(moduleConfig, binary, host.WithDeterminism())
+	if err != nil {
+		return nil, errors.New("unable to create module from config")
```

Comment:
Should we wrap the original error?
```suggestion
	if err != nil {
		return nil, fmt.Errorf("unable to create module from config: %w", err)
```